core/templates/pages/story-editor-page/editor-tab/story-node-editor.component.html

```diff
@@ -117,31 +117,45 @@
         <div class="e2e-test-add-chapter-outline">
           <div class="story-node-header oppia-mobile-collapsible-card-header" (click)="toggleChapterOutline()">
             <span>Chapter Outline</span>
-            <i class="fa fa-caret-down"
-               *ngIf="!chapterOutlineIsShown"
-               aria-hidden="true">
-            </i>
-            <i class="fa fa-caret-up"
-               *ngIf="chapterOutlineIsShown"
-               aria-hidden="true">
-            </i>
           </div>
-          <div class="chapter-card-inputs oppia-mobile-collapsible-card-content" *ngIf="chapterOutlineIsShown">
+          <div class="outline-finalized-container">
             Outline Finalized
             <i *ngIf="!outlineIsFinalized" (click)="finalizeOutline()" class="far fa-square md-18 e2e-test-finalize-outline" aria-label="Finalize outline"></i>
             <i *ngIf="outlineIsFinalized" (click)="unfinalizeOutline()" class="fas fa-check-square md-18" aria-label="Unfinalize outline"></i>
-            <div tabindex="-1" (blur)="toggleChapterOutlineButtons()" (click)="toggleChapterOutlineButtons()">
+          </div>
+          <div *ngIf="!chapterOutlineIsShown">
+            <div class="outline-content-container" *ngIf="!outlineEditViewIsShown">
+              <input class="chapter-outline-editor-content e2e-test-node-outline-editor-content e2e-test-rte"
+                     readonly
+                     (click)="makeChapterOutlineEditable()"
+                     placeholder="Open chapter outline editor.">
+              <label class="fas fa-pencil-alt oppia-icon" (click)="makeChapterOutlineEditable()"></label>
+            </div>
+            <div class="outline-content-container" *ngIf="outlineEditViewIsShown">
+              <button class="chapter-outline-editor-content e2e-test-node-outline-editor-content e2e-test-rte"
+                      (click)="makeChapterOutlineEditable()">{{outlineEditViewIsShown}}
+              </button>
+              <label class="fas fa-pencil-alt oppia-icon" (click)="makeChapterOutlineEditable()"></label>
+            </div>
+          </div>
+          <div class="chapter-card-inputs oppia-mobile-collapsible-card-content" *ngIf="chapterOutlineIsShown">
+            <div tabindex="-1" >
               <schema-based-editor id="storyNodeOutline"
                                    [schema]="OUTLINE_SCHEMA"
                                    [localValue]="editableOutline"
-                                   (localValueChange)="updateLocalEditableOutline($event)">
+                                   (localValueChange)="updateLocalEditableOutline($event);"
+                                   (click)="chapterOutlineButtonsAreShown=true; ">
```

Comment:
Do you still need the "click" action here? I don't think so, right?

Reply:
I'm fixing again, because there are CI tests problems.

Reply:
I fixed this. Then the fixed code passed CI tests.